src/peerctl/settings/__init__.py

```diff
@@ -29,6 +29,7 @@
 MIDDLEWARE += (
     # in fullctl    "fullctl.django.middleware.CurrentRequestContext",
     "fullctl.django.middleware.RequestAugmentation",
+    "fullctl.django.middleware.AutocompleteRequestPermsMiddleware",
```

Comment:
@jason20c i think this can probably be added in https://github.com/fullctl/fullctl/blob/main/src/fullctl/django/settings/__init__.py - given we will want this active for any service that sets up autocomplete endpoints, right?

Reply:
The problem is this needs to be at the end of the `MIDDLEWARE` list in order to work.
This is because the `RequestAugmentation` overwrites the perms.